aleph/logic/mapping.py

```diff
@@ -0,0 +1,113 @@
+import logging
+from banal import first
+from flask import request
+from werkzeug.exceptions import BadRequest
+from followthemoney import model
+from banal import hash_data, keys_values
+
+from aleph.core import archive, db
+from aleph.views.util import get_index_entity, get_session_id
```

Comment:
Logic layer should never import from `from aleph.views`, that's a sign of something being in the wrong place. In this case I'd say the `flush_mapping` and `load_mapping` functions want to live in the API.